lighthouse-core/test/audits/byte-efficiency/unused-javascript-test.js

```diff
@@ -56,134 +56,107 @@ function generateUsage(url, ranges, transferSize = 1000) {
+    assert.equal(inlineWaste.wastedBytes, 6000);
+    assert.equal(Math.round(inlineWaste.wastedPercent), 29);
   });
 
-  describe('audit_', () => {
-    const domain = 'https://www.google.com';
-    const scriptUnknown = generateUsage(domain, [[0, 3000, false]]);
-    const scriptA = generateUsage(`${domain}/scriptA.js`, [[0, 100, true]]);
-    const scriptB = generateUsage(`${domain}/scriptB.js`, [[0, 200, true], [0, 50, false]]);
-    const inlineA = generateUsage(`${domain}/inline.html`, [[0, 5000, true], [5000, 6000, false]]);
-    const inlineB = generateUsage(`${domain}/inline.html`, [[0, 15000, true], [0, 5000, false]]);
-    const recordA = generateRecord(`${domain}/scriptA.js`, 35000, 'Script');
-    const recordB = generateRecord(`${domain}/scriptB.js`, 50000, 'Script');
-    const recordInline = generateRecord(`${domain}/inline.html`, 1000000, 'Document');
-
-    it('should merge duplicates', async () => {
-      const context = {computedCache: new Map()};
-      const networkRecords = [recordA, recordB, recordInline];
-      const artifacts = {
-        JsUsage: [scriptA, scriptB, scriptUnknown, inlineA, inlineB],
-        devtoolsLogs: {defaultPass: networkRecordsToDevtoolsLog(networkRecords)},
-      };
-      const result = await UnusedJavaScript.audit_(artifacts, networkRecords, context);
-      assert.equal(result.items.length, 2);
-
-      const scriptBWaste = result.items[0];
-      assert.equal(scriptBWaste.totalBytes, 50000);
-      assert.equal(scriptBWaste.wastedBytes, 12500);
-      assert.equal(scriptBWaste.wastedPercent, 25);
-
-      const inlineWaste = result.items[1];
-      assert.equal(inlineWaste.totalBytes, 21000);
-      assert.equal(inlineWaste.wastedBytes, 6000);
-      assert.equal(Math.round(inlineWaste.wastedPercent), 29);
-    });
-
-    it('should augment when provided source maps', async () => {
-      const context = {
-        computedCache: new Map(),
-        options: {
-          // Default threshold is 1024, but is lowered here so that squoosh actually generates
-          // results.
-          // TODO(cjamcl): the bundle visualization feature will require most of the logic currently
-          // done in unused-javascript to be moved to a computed artifact. When that happens, these
-          // tests will go there, and the artifact will not have any thresholds (filtering will happen
-          // within the audits), so this test will not need a threshold. Until then, it does.
-          bundleSourceUnusedThreshold: 100,
+  it('should augment when provided source maps', async () => {
+    const context = {
+      computedCache: new Map(),
+      options: {
+        // Default threshold is 512, but is lowered here so that squoosh generates more
+        // results.
+        // TODO(cjamcl): the bundle visualization feature will require most of the logic currently
+        // done in unused-javascript to be moved to a computed artifact. When that happens, these
```

Comment:
isn't that what this is?

I'm a little unclear of what else needs to happen to remove this TODO if it's not this PR :)

Reply:
ya oops

Reply:
so moving this test to the computed artifact makes that file way more complicated. WDYT of keeping the test here, and only testing the source map features of the computed artifact indirectly via the audit test?

Reply:
yeah keeping the test here sounds fine to me, we've got the basics in place for testing the computed anyhow.

Reply:
just remove the TODO then :)